components/mpas-seaice/bld/namelist_files/namelist_defaults_mpassi.xml

```diff
@@ -76,11 +77,13 @@
 <config_initial_latitude_north ice_grid="oEC60to30v3wLI">75.0</config_initial_latitude_north>
 <config_initial_latitude_north ice_grid="ECwISC30to60E1r2">75.0</config_initial_latitude_north>
 <config_initial_latitude_north ice_grid="SOwISC12to60E2r4">85.0</config_initial_latitude_north>
+<config_initial_latitude_north ice_grid="ECwISC30to60E2r1">85.0</config_initial_latitude_north>
```

Comment:
Should this be 75.0 to be consistent w/ ECwISC30to60E1r2?

Reply:
@stephenprice I suppose this should be something we decide on, so thanks for pointing this out. I put this at +/- 85 to be consistent with the SORRM grid for v2 runs, which I was basing on discussions from v1 that the smaller disk was better in some way. I'm open to reverting this back to +/- 75, but I should redo the spun up ICs used for B-cases to be consistent with that.

Reply:
I'm not sure if it's important enough to redo initial conditions. Maybe the consensus is that the initial lats. for sea ice should be narrower going forward?

Reply:
@stephenprice thinking more on this, I think we should just stick to +/- 75. I can't recall a consensus reach on moving to +/- 85. Note these default sea ice ICs are only used for G-cases, as the B-cases point to 1 month restarts from a G-case. I just checked the `SOwISC12to60E2r4` restartFromG ICs I generated and those also used +/- 75, so I should make that correction for that grid in a separate PR.